numpy-stubs/__init__.pyi

```diff
@@ -390,6 +390,8 @@ class ndarray(Generic[_DType]):
     def __radd__(self, value: ndarray[_DType]) -> ndarray[_DType]: ...
     @overload
     def __radd__(self, value: _DType) -> ndarray[_DType]: ...
+    @overload
+    def __radd__(self, value: float) -> ndarray[_DType]: ...
```

Comment:
I think the return type should be float64?
```suggestion
    def __radd__(self, value: float) -> ndarray[float64]: ...
```

Reply:
It´s for the `float + array` case, so the return is also an array

Reply:
Hmm, I think that not necessarily... if `_DType` is `np.int64`, the return type will indeed be `np.float64`, but this would not be true for complex types, for example.